packages/ckeditor5-page-break/tests/pagebreakediting.js

```diff
@@ -57,7 +57,7 @@ describe( 'PageBreakEditing', () => {
 		setModelData( model, '[<pageBreak></pageBreak>]' );
 		const textNode = viewDocument.getRoot().getChild( 0 ).getChild( 0 );
```

Comment:
I don't know why this variable is called `textNode`? From what I see it holds an elemnt.